wsd/DocumentBroker.hpp

```diff
@@ -230,6 +230,8 @@ class DocumentBroker : public std::enable_shared_from_this<DocumentBroker>
         alertAllUsers("error: cmd=" + cmd + " kind=" + kind);
     }
 
+	void setLogLevel(const std::string& level);
```

Comment:
Documentation is helpful to others to understand what this is supposed to do. Also, the name is misleading. If this is supposed to set the *child* process's log level, it should be setChildLogLevel or setKitLogLevel. But regardless of the name, please document all new functions, even if their purpose seems obvious to you.